### PR TITLE
Add Bitbucket Plugin

### PR DESCRIPTION
**Pull request by EugenGanshorn**
## Contribution type

*Bug fix, new plugin, feature, ui etc.*

## Description of change

*Detailed description of change.*

### CODE REVIEW

File: src/Plugin/Bitbucket.php [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+     * @var Database
+     */
+    protected $pdo;
+
+    /**
+     * @return string
+     */
+    public static function pluginName()
+    {
+        return 'bitbucket';
```
Comment (corpsee) [inline src/Plugin/Bitbucket.php]: I saggest to rename plugin to `BitbucketNotify` (`bitbucket_notify`).

Comment (EugenGanshorn) [reply src/Plugin/Bitbucket.php]: i agree

**Comment by corpsee** [comment]
@EugenGanshorn :+1: I am ready to merge the changes when you are rename plugin. Thank you!

**Comment by EugenGanshorn** [comment]
> @EugenGanshorn 👍 I am ready to merge the changes when you rename plugin. Thank you!

i am done :-D

File: docs/en/plugins/bitbucket.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+Plugin Bitbucket
```
Comment (corpsee) [inline docs/en/plugins/bitbucket.md]: Bitbucket Notify.

File: docs/en/plugins/bitbucket.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+* **create_task_per_fail** [bool, optional] - Create a task per "failed" plugin.
+* **create_task_if_fail** [bool, optional] - Create one task if at least one plugin "failed".
+* **update_build** [bool, optional] - Update build status in Bitbucket.
+* **message** [string, optional] - Overwrite the default message.
+
+### Examples
+
+```yaml
+complete:
+    bitbucket:
```
Comment (corpsee) [inline docs/en/plugins/bitbucket.md]: `bitbucket_notify`.

File: docs/en/plugins/bitbucket.md (hunk 2 above)
Comment (EugenGanshorn) [reply docs/en/plugins/bitbucket.md]: thx, fixed

File: docs/en/plugins/bitbucket.md (hunk 3 above)
Comment (EugenGanshorn) [reply docs/en/plugins/bitbucket.md]: thx, fixed

**Comment by corpsee** [comment]
PR merged to `master` branch and will be released with version `1.1.0`.

**Pull request merged**